extensions/multiply-divide/README.md

```diff
@@ -0,0 +1,50 @@
+# General design
+
+**Extension State: Under Development**  
+**Requires: Base, VWI, CP2.0**  
+**CPUID Bit: CP2.3**
+
+# Overview
+
+This extension adds multiplication, division, and remainder instructions. 
+
+# Added Instructions
+
+These instructions are in the expanded calculation opcode section of instructions.
+
+A `U` at the front of an instruction indicates that it treats its operands as unsigned.
+An `S` indicates that the operands are treated as signed.
+
+### Opcode table
+
+| `C CCCC CCCC` | NAME     | Operation                                  | Flags  | Comment |
+|---------------|----------|--------------------------------------------|--------|---------|
+| `0 0001 0000` | `UDIV`   | `A ← A / B`                                | None   | (1)     |
+| `0 0001 0001` | `SDIV`   | `A ← A / B`                                | None   | (1)     |
+| `0 0001 0010` | `UREM`   | `A ← A % B`                                | None   | (1) (2) |
+| `0 0001 0011` | `SREM`   | `A ← A % B`                                | None   | (1) (2) |
```

Comment:
We should mention that these will cause interrupts/undefined behavior on division by zero.

Reply:
I missed that section completly the first two times I read these PRs 🙃